PSMSVirtualDeviceManager/StartPage/UCStartPage.vb

```diff
@@ -691,4 +696,32 @@ Public Class UCStartPage
     Private Sub Button_PsmsUpdateIgnore_Click(sender As Object, e As EventArgs) Handles Button_PsmsUpdateIgnore.Click
         Panel_PsmsxUpdate.Visible = False
     End Sub
+
+    Public Function CheckIfServiceRunning() As Integer
```

Comment:
Create an enum instead of integer. Its easier to read than numbers.

```suggestion
Enum ENUM_SERVICE_PROCESS_TYPE
    NONE,
    NORMAL,
    ADMIN
End Enum

    Public Function CheckIfServiceRunning() As ENUM_SERVICE_PROCESS_TYPE
```